docs/reference/UDS Core/distribution-support.md

```diff
@@ -12,6 +12,6 @@ UDS Core is a versatile software baseline designed to operate effectively across
 
 | Distribution    | Category               | Support Level                                                                                             |
 | --------------- | ---------------------- | --------------------------------------------------------------------------------------------------------- |
-| K3d/K3s, Amazon EKS | Tested                 | Supported Kubernetes distributions undergoing testing in CI environments.                                 |
+| K3d/K3s, Amazon EKS, Azure AKS | Tested                 | Supported Kubernetes distributions undergoing testing in CI environments.                                 |
```

Comment:
Could you also add `RKE2 (on AWS)` or something similar here since we have that in a nightly CI test now?